src/main/scala/org/broadinstitute/dsde/rawls/dataaccess/HttpExecutionServiceDAO.scala

```diff
@@ -30,4 +30,11 @@ class HttpExecutionServiceDAO( executionServiceURL: String )( implicit system: A
     val pipeline = addHeader(Cookie(authCookie)) ~> sendReceive ~> unmarshal[ExecutionServiceStatus]
     Await.result(pipeline(Get(url)),Duration.Inf)
   }
+
+  override def outputs(id: String, authCookie: HttpCookie): ExecutionServiceOutputs = {
+    val url = executionServiceURL + s"/workflow/${id}/outputs"
+    import system.dispatcher
```

Comment:
I've noticed that this particular import ( ```system.dispatcher``` ) consistently shows up in the middle of functions. I suspect it's some magic thing related to Akka/Spray.